Patch by Martin Vaeth <martin at mvath.de> from May 10, 2015
--- 1/Portage.pm
+++ 2/Portage.pm
@@ -938,7 +938,7 @@
 			for my $flag (@flags) {
 				my $state = $flag =~ s/^-// || 0;
 
-				_add_temp($flag, "global");
+				_add_temp($flag, "local");
 				_add_temp($flag, $pkg);
 
 				if ($state) {
